libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```
@@ -182,7 +281,28 @@ export const DatePicker: React.FC<React.PropsWithChildren<DatePickerProps>> = ({
+                setStartDate(startDay)
+                endDay && setEndDate(endDay)
+
+                handleChange &&
+                  startDay &&
+                  endDay &&
+                  handleChange(startDay, endDay)
+              }}
+              ranges={ranges}
+              highlightWeekends={highlightWeekends}
+              displaySelectInput={displaySelectInput}
+              children={props.children}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)